pulsar-broker/src/main/java/org/apache/pulsar/broker/service/PulsarStats.java

```diff
@@ -183,7 +183,7 @@ public synchronized void updateStats(
                         topicStatsStream.endObject();
                     });
 
-                    brokerStats.bundleCount += bundles.size();
+                    brokerStats.bundleCount += (int) bundles.size();
```

Comment:
nit: do we need `(int)`?  seems like bundles.size() is already long.

Reply:
Good point. I initially did the cast and then moved to long.. Fixed.